practicals/Recommender_Systems.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>What does it say about you as the reader and your preferences? Could there be</p><p>other people like you?&nbsp;-&gt; looks like there's an extra line break here</p><p> </p><p>Also, I cannot see most of the images in the prac, so I am unable to review them. I am supposing that they are in fact present.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Good catch. Fixed it.</p><p> </p><p>Also fixed the images. For some it wasn't reading them. I've also made them local to the notebook now.</p>